src/dashboard/src/components/filesystem_ajax/views.py

```diff
@@ -928,22 +928,44 @@ def download_fs(request):
         raise django.http.Http404
 
 
-def download_by_uuid(request, uuid):
+def preview_by_uuid(request, uuid):
+    """Simple wrapper to the download_by_uid function to create a route
+    through the code to enable the preview of a file in the browser, not
+    automatically download.
     """
-    Download a file from the Storage Service, given its UUID.
+    return download_by_uuid(request, uuid, True)
 
-    This view will stream the response directly from the storage service, so, unlike download_ss, this will work even if the Storage Service is not accessible to the requestor.
 
-    Returns 404 if a file with the requested UUID cannot be found, and 400 if the storage service fails to retrieve the record.
+def download_by_uuid(request, uuid, preview_file=False):
+    """Download a file from the Storage Service, given its UUID.
+
+    This view will stream the response directly from the storage service,
+    so, unlike download_ss, this will work even if the Storage Service is
+    not accessible to the requestor.
+
+    Returns 404 if a file with the requested UUID cannot be found, and
+    400 if the storage service fails to retrieve the record.
```

Comment:
It doesn't seem right that we're sending a 400 code to the browser (aka "your request is broken") when SS doesn't return a 200. It's good that we're documenting this, we should probably change it at some point. Maybe a TODO note here would be wise.

Reply:
I was about to log this in a Github issue, rather than a `todo` but I am looking at the response, I can't see the 400 - what do you think? The docstring is wrong?

Reply:
Have logged a ticket here. https://github.com/artefactual/archivematica/issues/994 Will update the docstring to be more specific about how it's returning here. It's a little ambiguous in that it actually returns whatever stream from storage service returns - I'm guessing from a previous refactor.